go/vt/sqlparser/ast.go

```diff
@@ -586,6 +587,21 @@ func (node *Other) WalkSubtree(visit Visit) error {
 	return nil
 }
 
+// Admin represents a REPAIR, OPTIMIZE, or TRUNCATE statement.
```

Comment:
I might make this more generic, since the list of query types parsed this could easily change over time:  `// Admin represents a misc statement that relies on ADMIN privileges, such as TRUNCATE`